queries/ecma/highlights.scm

```diff
@@ -175,9 +175,9 @@
 "{" @punctuation.bracket
 "}" @punctuation.bracket
 
-(template_substitution
-  "${" @punctuation.special
-  "}" @punctuation.special) @none
+((template_substitution ["${" "}"] @punctuation.special) @none
+ ; Substitutios should have more priority over injections.
+ (#set! "priority" 105))
```

Comment:
But what is then the objective of the injection when it's now shown because it has a lower priority?

Reply:
No sure I understand your question, this only increase the priority of the substitutions (`{foo}`), the injection is still processed (as you can see on the screenshot).

Reply:
but can you fix the typo?

Reply:
done!